clio-integration-test/src/it/scala/org/broadinstitute/clio/integrationtest/EnvIntegrationSpec.scala

```diff
@@ -50,16 +50,20 @@ abstract class EnvIntegrationSpec(env: String)
     rootPathForBucketInEnv(env, storageScopes)
 }
 
-/** The integration spec that runs against Clio in dev. */
-class DevEnvIntegrationSpec extends EnvIntegrationSpec("dev") with IntegrationSuite
+/** The integration specs that run against Clio in dev. */
+class DevEnvBasicSpec extends EnvIntegrationSpec("dev") with BasicTests
```

Comment:
Ah, this will break the regex in the nightly jenkins job (it expects `*DevEnvIntegrationSpec`). I like these less-verbose names though, so I'd say just update the job in dsp-jenkins to match the new pattern.

Reply:
Ready to merge dsp-jenkins change.